client/state/selectors/is-user-registration-days-within-range.js

```diff
@@ -1,35 +1,28 @@
-/** @format */
-
-/**
- * External dependencies
- */
-import moment from 'moment';
-
 /**
  * Internal dependencies
  */
 import { getCurrentUserDate } from 'state/current-user/selectors';
 
+const DAY_IN_MS = 1000 * 60 * 60 * 24;
+
 /**
  * Returns true if the number of days the current user has been registered for falls within the specied range of values.
  *
  * @param {Object} state Global state tree
- * @param {Object} momentInTime Current moment for determination of elapsed days since registration
+ * @param {Date} refDate Date for determination of elapsed days since registration
```

Comment:
Note that while we've changed the interface for this method here, by moving from a `moment` to a `Date`, there appears to be a single usage of this selector in the codebase, and it's passing in a `null` on this parameter. Only the tests needed updating.